cpu/sam0_common/periph/i2c.c

```diff
@@ -92,23 +92,6 @@ void i2c_init(i2c_t dev)
 
     /* I2C using CLK GEN 0 */
     sercom_set_gen(bus(dev),i2c_config[dev].gclk_src);
-#if defined(CPU_FAM_SAML21) || defined(CPU_FAM_SAMR30)
```

Comment:
what the heck? We don't want to change something in sam0 when working on cc2560

Reply:
Sorry, from another pr... git issues.